### PR TITLE
SDI-333 Changes to event handling to improve latency

### PR DESCRIPTION
**Pull request by steverendell**
results of project rename script

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsprisonerevents/config/HmppsPrisonerEventsExceptionHandler.kt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package uk.gov.justice.digital.hmpps.hmppstemplatepackagename.config
+package uk.gov.justice.digital.hmpps.hmppsprisonerevents.config
```
Comment (andymarke) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsprisonerevents/config/HmppsPrisonerEventsExceptionHandler.kt:1]: Suggest removing `hmpps` from `hmppsprisonerevents` else you have hmpps twice

**Review by andymarke — APPROVED** [review]
lgtm

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsprisonerevents/config/HmppsPrisonerEventsExceptionHandler.kt (hunk 1 above)
Comment (steverendell) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsprisonerevents/config/HmppsPrisonerEventsExceptionHandler.kt:1]: good point

**Review by andymarke — APPROVED** [review]
LGTM

**Pull request merged**